docs/getting-started/docker-playgrounds/docker-compose.yaml

```diff
@@ -0,0 +1,50 @@
+version: "2.1"
```

Comment:
So this file is temporary or we will include a download link?

Reply:
Yes, I thought we'll only embedded into the documentation. Download would of course also work. What do you think?

Reply:
Ideally, I would like to, but if we can't get it into the CI pipeline I'm sure it will quickly and quietly get stale and go out of sync.

Reply:
Not sure I completely understand your last comment.

In the long run we will only need the docker-compose file (when Flink Configuration can be injected into the containers by environment variables). I don't think there is any way around manually maintaining this file. Once this is only one file, it is also easier to simply put it somewhere to download. 

One advantage of embedding it into the docs is, that it we can template things like the Flink version.

Reply:
Alternatively, I can of course put these files into a separate repository (e.g. knaufk/flink-cluster-playground). This would be the easiest for the user, but is of course also not optimal.